src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -0,0 +1,159 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Threading;
+using Xunit;
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
+
+namespace System.Device.Gpio.Tests
+{
+    public abstract class GpioControllerTestBase
+    {
+        private const int LedPin = 18;
```

Comment:
For now, these pinNumbers are hardcoded, but the idea is to make them instead configurable so that they will look into a configuration file (if present) to see which value should they use, and fallback to a provided default value.